doc/nrf/release_notes.rst

```diff
@@ -17,7 +17,7 @@ Important issues that are discovered after a release is tagged are listed in the
    :maxdepth: 1
```

Comment:
For this release, these links will stay, because there was no final decision about this (hopefully next week). If we decide to deprecate the wiki, we will add a link pointing to the new known issues page and the "Changes in vX.Y.99" page.